var/spack/repos/builtin/packages/python/package.py

```diff
@@ -135,7 +135,7 @@ class Python(AutotoolsPackage):
         # a Mac.
         depends_on('libuuid', when='+uuid')
 
-    patch('tkinter.patch', when='@:2.8,3.3: platform=darwin')
+    patch('tkinter.patch', when='@:2.8,3.3:3.7 platform=darwin')
```

Comment:
@tgamblin already noticed this patch problem, the patch has been merged upstream.